app/controllers/concerns/va_inherited_proofing_concern.rb

```diff
@@ -0,0 +1,27 @@
+# For VA-initiated inherited proofing.
+# https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/Inherited%20Proofing/MHV%20Inherited%20Proofing/inherited-proofing-interface.md
+module VaInheritedProofingConcern
+  extend ActiveSupport::Concern
+
+  def va_inherited_proofing?
```

Comment:
I do like that these are prefixed, but it does signal to me that perhaps there should be a service or something like that that could wrap them?

Reply:
Voiding name clashes :S. I thought the same, except the idea was to only handle the incoming IP requests in concerns (see prev comments). Again, would be glad to defer, say the word :)